<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DownloadClass</class>
 <widget class="QMainWindow" name="DownloadClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Download</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout" stretch="0,1">
    <property name="spacing">
     <number>5</number>
    </property>
    <property name="leftMargin">
     <number>9</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>9</number>
    </property>
    <property name="bottomMargin">
     <number>9</number>
    </property>
    <item>
     <widget class="QWidget" name="toolWidget" native="true">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QToolButton" name="addBtn"/>
       </item>
       <item>
        <widget class="QToolButton" name="stopBtn"/>
       </item>
       <item>
        <widget class="QToolButton" name="stopAllBtn"/>
       </item>
       <item>
        <widget class="QToolButton" name="delBtn"/>
       </item>
       <item>
        <widget class="QToolButton" name="delAllBtn"/>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QTreeView" name="treeView"/>
      <widget class="QTableView" name="tableView"/>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuTasks">
    <property name="title">
     <string>Tasks</string>
    </property>
    <addaction name="actionAdd_new_down"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionStop_down"/>
    <addaction name="actionRemove"/>
    <addaction name="actionDownload_now"/>
    <addaction name="actionRedownload"/>
   </widget>
   <widget class="QMenu" name="menuDownloads">
    <property name="title">
     <string>Downloads</string>
    </property>
    <addaction name="actionPause_All"/>
    <addaction name="actionStop_All"/>
    <addaction name="separator"/>
    <addaction name="actionDelete_All_Completed"/>
    <addaction name="separator"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="actionLanguage"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionCheck_for_updates"/>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuTasks"/>
   <addaction name="menuFile"/>
   <addaction name="menuDownloads"/>
   <addaction name="menuView"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionAdd_new_down">
   <property name="text">
    <string>Add new download</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionStop_down">
   <property name="text">
    <string>Stop download</string>
   </property>
  </action>
  <action name="actionRemove">
   <property name="text">
    <string>Remove</string>
   </property>
  </action>
  <action name="actionDownload_now">
   <property name="text">
    <string>Download now</string>
   </property>
  </action>
  <action name="actionRedownload">
   <property name="text">
    <string>Redownload</string>
   </property>
  </action>
  <action name="actionLanguage">
   <property name="text">
    <string>Language</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionCheck_for_updates">
   <property name="text">
    <string>Check for updates</string>
   </property>
  </action>
  <action name="actionPause_All">
   <property name="text">
    <string>Pause All</string>
   </property>
  </action>
  <action name="actionStop_All">
   <property name="text">
    <string>Stop All</string>
   </property>
  </action>
  <action name="actionDelete_All_Completed">
   <property name="text">
    <string>Delete All Completed</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="Download.qrc"/>
 </resources>
 <connections/>
</ui>
